rest/taskrouter/statistics/workflow/cumulative/example-1.6.x.py

```diff
@@ -0,0 +1,16 @@
+# Download the Python helper library from twilio.com/docs/python/install
+from twilio.rest import Client
+
+# Your Account Sid and Auth Token from twilio.com/user/account
+account_sid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+auth_token = "your_auth_token"
+workspace_sid = "WSXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+workflow_sid = "WWXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+
+client = Client(account_sid, auth_token)
+
+statistics = client.taskrouter.workspaces(workspace_sid) \
+    .workflows(workflow_sid).cumulativestatistics().fetch()
```

Comment:
@jwitz10, you may want to use `workers_cumulative_statistics`. https://github.com/twilio/twilio-python/blob/df0b500811ff9567d27aaa6c2099c360e2506f46/twilio/rest/taskrouter/v1/workspace/workflow/__init__.py#L386.

I found this method name quite verbose, what do you think about asking the Yoyodine team for something closer to what you have: `cumulative_statistics`?

Reply:
Agreed.